### PR TITLE
Projects section

### PR DESCRIPTION
**Pull request by VictoriaKiselova**

### CODE REVIEW

File: src/partials/projects.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
```
Comment (laposhko) [inline src/partials/projects.html]: body is in index.html, you need only create section

File: src/partials/projects.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                                </p>
+                                <a class="projects-link" href="https://github.com/laposhko/NotWizards.YET"
+                                    target="_blank">See
+                                    project</a>
+                            </div>
+                        </div>
+                        <div class="projects-image-box">
+                            <picture>
+                                <source media="(min-width: 1440px)"
+                                    srcset="../img/projects/projects-petlove-desktop.jpg 1x, ../img/projects/projects-petlove-desktop@2x.jpg 2x"
```
Comment (laposhko) [inline src/partials/projects.html]: such path will not work in GitHub Pages, please change to ./img/projects/projects-petlove-desktop.jpg

**Review by laposhko — APPROVED** [review]

File: src/partials/projects.html (hunk 2 above)
Comment (VictoriaKiselova) [reply src/partials/projects.html]: +

File: src/partials/projects.html (hunk 1 above)
Comment (VictoriaKiselova) [reply src/partials/projects.html]: ok

**Pull request merged**